src/main/java/ch/njol/skript/events/EvtMove.java

```diff
@@ -42,47 +41,89 @@ public class EvtMove extends SkriptEvent {
 			events = CollectionUtils.array(PlayerMoveEvent.class, EntityMoveEvent.class);
 		else
 			events = CollectionUtils.array(PlayerMoveEvent.class);
-
-		Skript.registerEvent("Move", EvtMove.class, events, "%entitydata% (move|walk|step)")
-			.description("Called when a player or entity moves.",
-				"NOTE: Move event will only be called when the entity/player moves position, not orientation (ie: looking around).",
-				"NOTE: These events can be performance heavy as they are called quite often.",
-				"If you use these events, and later remove them, a server restart is recommended to clear registered events from Skript.")
-			.examples("on player move:",
-				"\tif player does not have permission \"player.can.move\":",
-				"\t\tcancel event",
-				"on skeleton move:",
-				"\tif event-entity is not in world \"world\":",
-				"\t\tkill event-entity")
-			.requiredPlugins("Paper 1.16.5+ (entity move)")
-			.since("2.6");
+		Skript.registerEvent("Move / Rotate", EvtMove.class, events,
+				"%entitydata% (move|walk|step|rotate:(look[ing] around|rotate))",
+				"%entitydata% (move|walk|step) or (look[ing] around|rotate)")
+				.description(
+						"Called when a player or entity moves or rotates their head.",
+						"NOTE: Move event will only be called when the entity/player moves position, not orientation (ie: looking around). Use the keyword 'rotate' instead.",
+						"NOTE: These events can be performance heavy as they are called quite often.",
+						"If you use these events, and later remove them, a server restart is recommended to clear registered events from Skript.")
+				.examples(
+						"on player move:",
+							"\tif player does not have permission \"player.can.move\":",
+								"\t\tcancel event",
+						"on skeleton move:",
+							"\tif event-entity is not in world \"world\":",
+								"\t\tkill event-entity",
+						"on player rotate:",
+							"send action bar \"You are currently looking around!\" to player")
+				.requiredPlugins("Paper 1.16.5+ (entity move)")
+				.since("2.6, INSERT VERSION (rotate)");
 	}
 
 	private EntityData<?> type;
 	private boolean isPlayer;
+	private boolean isRotate;
+	private Move moveType;
+
+	private enum Move {
+
+		POSITION("move"),
+		ORIENTATION("rotate"),
+		POSITION_OR_ORIENTATION("move or rotate");
```

Comment:
Do you think it might be better to use the same terminology in the enum and the name?
Then you could get the string name from the enum name.

Reply:
Answered in skUnity Discord that this is just a suggestion and the current implemention is fine. If any members preferred this way do let me know!